packages/domain-picker/test/suggestion-item.tsx

```diff
@@ -336,68 +120,31 @@ describe( 'check conditional elements render correctly', () => {
 		// use `queryBy` to avoid throwing an error with `getBy`
 		expect( screen.queryByTestId( 'info-tooltip' ) ).toBeFalsy();
 	} );
+	/*eslint-enable*/
 
-	it( 'renders recommendation badge if given prop isRecommended true', async () => {
-		const testRequiredProps = {
-			domain: 'testdomain.com',
-			cost: '€12.00',
-			railcarId: 'id',
-			isRecommended: true,
-		};
+	it( 'should render a recommendation badge if given prop isRecommended true', async () => {
+		renderComponent( { cost: '€12.00', isRecommended: true } );
 
-		render(
-			<SuggestionItem { ...testRequiredProps } onSelect={ jest.fn() } onRender={ jest.fn() } />
-		);
-
-		expect( screen.getByText( /Recommended/i ) ).toBeTruthy();
+		expect( screen.getByText( /Recommended/i ) ).toBeInTheDocument();
 	} );
 
-	it( 'renders the cost if given prop of cost with a value', async () => {
-		const testRequiredProps = {
-			domain: 'testdomain.com',
-			cost: '€12.00',
-			railcarId: 'id',
-			isRecommended: true,
-		};
-
-		render(
-			<SuggestionItem { ...testRequiredProps } onSelect={ jest.fn() } onRender={ jest.fn() } />
-		);
+	it( 'should render the cost if given prop of cost with a value', async () => {
+		renderComponent( { cost: '€12.00' } );
 
-		expect( screen.queryByText( /Renews at: €12.00/i ) ).toBeTruthy();
+		expect( screen.getByText( /Renews at: €12.00/i ) ).toBeInTheDocument();
```

Comment:
`queryBy*` should only be used while checking for non-existing elements (see: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)